### PR TITLE
[Testing] TF2-ish Critical Hits 1.0.0.0

### PR DESCRIPTION
**Pull request by Berna-L**
Love dealing critical hits in Final Fantasy XIV, but you feel there's some... **_oomph_** missing?

Worry no more! With the TF2-ish Critical Hits plugin you can (almost) reproduce the experience of dealing Critical Hits and Mini Crits in the #1 hat simulator, Team Fortress 2!

### CODE REVIEW

**Comment by goaaats** [comment]
bleatbot, approve

File: testing/live/Tf2CriticalHits/manifest.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [plugin]
-repository = "https://github.com/Berna-L/ffxiv-tf2-crit-plugin"
+repository = "https://github.com/Berna-L/ffxiv-tf2-crit-plugin.git"
 commit = "f6512969d133ca160b0053bbdca08c4858eae311"
 owners = ["Berna-L"]
 project_path = "/TF2CriticalHitsPlugin"
```
Comment (MidoriKami) [inline testing/live/Tf2CriticalHits/manifest.toml:5]: the slash is implied, this might complain

Comment (Berna-L) [reply testing/live/Tf2CriticalHits/manifest.toml]: Removed it, thanks!

**Comment by NotNite** [comment]
....that's unique! Likely not your fault though. I believe there are some poorly timed server hiccups.

**Comment by Berna-L** [comment]
bleatbot, rebuild

**Comment by MidoriKami** [comment]
This is the thing with naudio and winforms right? What was the workaround to make that build in d17?

**Comment by Berna-L** [comment]
> This is the thing with naudio and winforms right? What was the workaround to make that build in d17?

This? https://github.com/goatcorp/DalamudPluginsD17/pull/112#issuecomment-1222769995

I looked at PeepingTom's current code and yeah, it uses only those two packages and another algorithm to play sounds: https://git.anna.lgbt/ascclemens/PeepingTom/src/branch/main/Peeping%20Tom/TargetWatcher.cs#L162

I guess I can use that as a base for my plugin.

**Comment by NotNite** [comment]
That error is pretty funny - my guess is it's because your json file has mismatched capitalization, and since the build system is Linux, it's affected by it (ext4 is case sensitive, NTFS isn't)

**Comment by Berna-L** [comment]
Yeah, I'm trying to fix it but Git (on Windows at least) doesn't detect capitalization changes as something to be stageable... Going to have to do a workaround.

**Comment by NotNite** [comment]
You should be able to use `git mv` if memory serves me.

**Comment by NotNite** [comment]
Oh, missed this too :P - the folder name in this repo needs to match the internal name of your plugin

**Comment by reiichi001** [comment]
> > This is the thing with naudio and winforms right? What was the workaround to make that build in d17?
> 
> This? [#112 (comment)](https://github.com/goatcorp/DalamudPluginsD17/pull/112#issuecomment-1222769995)
> 
> I looked at PeepingTom's current code and yeah, it uses only those two packages and another algorithm to play sounds: https://git.anna.lgbt/ascclemens/PeepingTom/src/branch/main/Peeping%20Tom/TargetWatcher.cs#L162
> 
> I guess I can use that as a base for my plugin.

You can also add `<EnableWindowsTargeting>true</EnableWindowsTargeting>` to the csproj if you're using wave output instead of directsound and that should work.

**Comment by IC-Berna-PTB** [comment]
Whew, finally! Many thanks for your help with the build!

**Comment by Berna-L** [comment]
Whew, finally! Many thanks with the build process!

**Comment by reiichi001** [comment]
This plugin is now in the plugin review queue; we hope to get back to you soon!

**Comment by philpax** [comment]
We're happy to go ahead, but we've noticed that there aren't any sounds included. Is that intentional?

**Comment by Berna-L** [comment]
Yes. I can make that more clear in the full description if needed.

**Review by philpax — APPROVED** [review]

**Comment by philpax** [comment]
Congratulations on your first plugin!

As a first-time contributor, if you're in the `XIVLauncher & Dalamud` Discord server and would like to get access to the private developer channels for plugin feedback and others, please let us know your Discord handle and we'll grant you the role.

**Comment by Berna-L** [comment]
@philpax Thanks! My handle is Berna#4737.

**Pull request merged**